Makefile

```diff
@@ -145,7 +153,10 @@ else
 endif
 .PHONY: souffle
 souffle:
-	if ! command -v souffle; then \
+ifeq ($(INSTALL_SOUFFLE),false)
+	$(warning "Skipping Souffle installation. Set INSTALL_SOUFFLE=true to install.")
```

Comment:
I assume you are trying to optionally disable the Souffle installation, but the implementation below should have handled that. Did you get [this error](https://github.com/oracle/macaron/pull/841/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L167) on your macOS?